### PR TITLE
Add higher rate limit for staff and superusers to DRF endpoints

### PR DESCRIPTION
**Pull request by bbaker6225**

### CODE REVIEW

File: credentials/apps/api/v2/tests/test_views.py [hunk 1]
```diff
@@ -506,6 +506,17 @@ def assert_throttling_log_correct(self, log_capture, view_set):
             )
         )
 
+    def hit_credential_view_rate_limit(self, user, scope, list_path):
+        """ Helper for just hitting, not exceeding the credential view  limit. """
```
Comment (BenjiLee) [inline credentials/apps/api/v2/tests/test_views.py]: super nit: You have 2 spaces between `view  limit`

File: credentials/apps/api/v2/tests/test_views.py [hunk 2]
```diff
@@ -506,6 +506,17 @@ def assert_throttling_log_correct(self, log_capture, view_set):
             )
         )
 
+    def hit_credential_view_rate_limit(self, user, scope, list_path):
```
Comment (BenjiLee) [inline credentials/apps/api/v2/tests/test_views.py]: I think there is potential to make this method more generic to be used by `test_grade_view_throttling` as well.

additionally, we could move the note
```
         Note: There is a potential for this test to be flaky in the case the
         endpoint we are testing slows down over time and/or the rate limit is
         increased.
```
into this helper function

**Comment by BenjiLee** [comment]
@bbaker6225 Done with pass. Looks good overall

**Comment by bbaker6225** [comment]
Ready for the next round of review.  Will squash and rebase once approved

File: credentials/apps/api/v2/tests/test_views.py [hunk 3]
```diff
@@ -506,39 +506,43 @@ def assert_throttling_log_correct(self, log_capture, view_set):
...
+        Helper for just hitting, not exceeding the scope's rate limit.
 
-        Note: There is a potential for this test to be flaky in the case the
-        endpoint we are testing slows down over time and/or the rate limit is
-        increased.
+        Note: There is a potential for any test using this helper to be flaky
+        in the case the endpoint we are testing slows down over time and/or the
+        rate limit is increased.
         """
+        self.authenticate_user(user)
```
Comment (BenjiLee) [inline credentials/apps/api/v2/tests/test_views.py]: Could we authenticate the user outside of this helper?

Comment (bbaker6225) [reply credentials/apps/api/v2/tests/test_views.py]: Definitely, good point

**Pull request merged**